### ISSUE TITLE
[Deque Analysis] Bitstream Embargo Date Picker has several color contrast issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1159]
## Deque Analysis Summary

On the Submission form, the Embargo Date Picker / Fields have several color contrast issues

https://demo7.dspace.org/submit -> Upload a File -> Edit File & look for the embargo fields

1. Placeholder text has insufficient color contrast when date fields are disabled. Element has insufficient color contrast of 3.77 (foreground color: `#777777`, background color: `#e9ecef`)
    * Deque issue ticket IDs: 470012
2. Popup date picker (calendar widget) has several issues.
     * Date at top (e.g Apr, 2021) Foreground: `#777777`, Background: `#FFFFFF`, The contrast ratio is: just below 4.5:1 (4.478:1)
     * Days of week (e.g. Mo, Tu, We, Th, Fr, Sa, Su) Foreground: `#207698`, Background: `#E8EBF3`, The contrast ratio is: 4.3:1
     * Previous and next month dates (e.g. 1, 2, 3, etc.) Foreground: `#959595`, Background: `#FFFFFF`, The contrast ratio is: just below 3:1 (2.995:1)
     * Deque issue ticket IDs: 470017

## Recommended Fix
 
Use the color picker to select a better foreground/background color with a minimum of 4.5:1 contrast.  https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by tdonohue** [comment]
@atarix83 : assigning this to you for your analysis/feedback as it's all about the Embargo features on the Submission form.  (As a sidenote, if we cannot get the embargo "date picker" to meet our accessibility guidelines, I'm also OK with removing it in favor of a manual date entry. Just let me know what you think is most reasonable.)

**Comment by michdyk** [comment]
@tdonohue Could you assign me to this issue? My team and I have investigated the accessibility of the system recently, so I guess it will be great to have done this task until we are fresh after our initial analysis.

**Comment by michdyk** [comment]
@tdonohue please assign my colleague @Dawnkai to this task. He is going to provide the PR for this issue. Also when it will be accepted, I kindly ask to give credits for this bug fix contribution to PCG Academia. Thanks
